File: Java/Ejercicio/src/Ejercicio6.java
package Ejercicio.src;

import java.util.Scanner;

public class Ejercicio6 {
    public static void main(String[] args) throws Exception {
        Scanner keyboard = new Scanner(System.in);
        System.out.println("Please enter the value in pesos that you want to convert");
        double pesoArg = keyboard.nextInt();
        double dolar = 140;
        double euro = 120;
        double real = 12;
        double valueDolar = pesoArg / dolar;
        double valueEuro = pesoArg / euro;
        double valueReal = pesoArg / real;

        System.out.println("THE VALUE OF " + pesoArg + " PEOS ITS CORRESPONDING VALUE IN");
        System.out.println("DOLAR = " + valueDolar + "\n");
        System.out.println("EURO = " + valueEuro + "\n");
        System.out.println("REAL = " + valueReal + "\n");

        keyboard.close();
    }
}

File: Javascript/Ejercicio/Ejercicio6.js
    
      process.stdout.write("Please enter the value in pesos that you want to convert");
      process.stdin.on('data',async function(data1) {
        const pesoArg = await parseFloat(data1.toString());
        const dolar = 140;
        const euro = 120;
        const real = 12;
        const valueDolar = pesoArg / dolar;
        const valueEuro = pesoArg / euro;
        const valueReal = pesoArg / real;
        process.stdout.write(`THE VALUE OF ${pesoArg} PEOS ITS CORRESPONDING VALUE IN`);
        process.stdout.write(`DOLAR = ${valueDolar} \n`);
        process.stdout.write(`EURO = ${valueEuro} \n`);
        process.stdout.write(`REAL = ${valueReal} \n`);
        process.exit();
    });
    